Makefile

```diff
@@ -7,6 +7,7 @@ build:
 test:
 	flake8 --exclude lib
 	python3 -m unittest tests/*.py
+	pip3 install -r test_requirements.txt
```

Comment:
Need to run the `pip install` before `flake8` or `unittest` to ensure the required libs are installed before they are required.

Build is currently failing in travis because it hasn't loaded the dependencies before running them,